pages/dataservices/[did].vue

```diff
@@ -170,6 +169,26 @@
                     </a>
                   </div>
                 </dd>
+                <dt class="text-gray-plain font-bold">
+                  {{ $t('Publics éligibles') }}
+                </dt>
+                <dd
+                  v-if="dataservice.access_type === 'restricted'"
+                  class="p-0"
+                >
+                  <ul class="list-none p-0 space-y-1 m-0">
+                    <template
+                      v-for="audience in (['local_authority_and_administration', 'company_and_association', 'private'] as Array<DataserviceAccessAudienceType>)"
+                      :key="audience"
+                    >
+                      <DataservicesAccessAudienceCondition
+                        v-if="dataservice.access_audiences[audience]"
+                        :condition="dataservice.access_audiences[audience]"
+                        :audience
+                      />
```

Comment:
This doesn't work since access_audiences is an array of object not a record?

Reply:
`toForm` and `toApi` are handling the transformation from array to object e.g. https://github.com/datagouv/cdata/pull/486/files#diff-c9645de8c21f9d394386f3f8ff20e003f671de3308061aa6fc11a23392de02e6R16

Reply:
So the type is wrong and I do not see the list in my machine…